src/Types.hs

```diff
@@ -27,38 +27,63 @@ import qualified Data.Csv as CSV
 
 -- | Globally Unique ID (for reference in the database).  (FIXME: should we have different id types
 -- for different object types?)
-type GUID = Integer
+newtype GUID a = GUID Integer
```

Comment:
Also, I think we've just agreed it's not globally unique (and MetaInfo is inlined in the schema). Am I right?